light_node/src/configuration.rs

```diff
@@ -676,20 +676,26 @@ fn pool_cfg(
     args: &clap::ArgMatches,
     pool_name_discriminator: &str,
 ) -> TezosApiConnectionPoolConfiguration {
+    // Adjust this so that it properly matches the command-line argument.
```

Comment:
I would suggest update constants values instead of constructing them in run-time.

Reply:
Done. Better now?